src/Microsoft.Health.SqlServer/Features/Schema/SchemaJobWorker.cs

```diff
@@ -90,6 +90,12 @@ public async Task ExecuteAsync(SchemaInformation schemaInformation, string insta
             {
                 // this could happen during schema initialization until base schema is not executed so can be ignored
             }
+            // Error: "Can not connect to the database in its current state". This error can be for various DB states (recovering, inacessible) but we assume that our DB will only hit this for Inaccessible state
+            catch (SqlException ex) when (ex.ErrorCode == 40925)
+            {
+                // this can happen when a user has misconfigured CMK for > 30 min, which results in the DB having a status of "Inacessible".
+                _logger.LogInformation(ex, "The SQL database cannot be accessed in its current state.");
```

Comment:
Warning?

Reply:
I've been logging all of the CMK errors as Information so far - I know DRIs sometimes will check the warnings summary in the Exceptions dashboard and I don't want this to be anything people would need to investigate, since it is not our fault